### PR TITLE
Adding file for larex preamble file

### PR DESCRIPTION
**Pull request by NiraliJoshi17**
Adding a preamble file as a support-files.tex and define a rule for this in dune file.

### CODE REVIEW

**Comment by jonludlam** [comment]
Hi @NiraliJoshi17, thanks for this! It's looking good so far, you've added the latex file as an OCaml module. The next steps are to install it as part of the `support-files` CLI sub command, as suggested in issue #588 

Also you'll need to reformat the dune file as it's causing the CI to fail. To do this, install `ocamlformat.0.19.0` and run

```
dune build @fmt 
dune promote
```

and commit the results.

**Comment by EmileTrotignon** [comment]
Is this still relevant ?
@NiraliJoshi17

File: src/odoc/etc/support-files.tex [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    keywords={[2]{val,initializer,nonrec}}, keywordstyle={[2]\ocamlkeyword},
+    belowskip=0\baselineskip,
+    upquote=true,
+    literate={'"'}{\textquotesingle "\textquotesingle}3
+    {'\\"'}{\textquotesingle \textbackslash"\textquotesingle}4,
+  }
+  }{}
+  \newcommand{\inputchapter}[1]{\chapter{#1}
+    \input{#1}}
+ \begin{document}
```
Comment (Julow) [inline src/odoc/etc/support-files.tex:70]: That part seems dubious. The preamble shouldn't be a document.

**Comment by jonludlam** [comment]
I'm going to close this for now. When we do a support-files like command for latex we can always come back to take a look at this PR.

**Pull request closed without merging**